integration-tests/integration_test.go

```diff
@@ -430,7 +432,7 @@ func (s *RepeatedNetworkFlowTestSuite) SetupSuite() {
 	s.clientIP, err = s.getIPAddress("nginx-curl")
 	s.Require().NoError(err)
 
-	totalTime := (s.sleepBetweenCurlTime * s.numIter + s.sleepBetweenIterations) * s.numMetaIter + s.afterglowPeriod + 10
+	totalTime := (s.sleepBetweenCurlTime*s.numIter+s.sleepBetweenIterations)*s.numMetaIter + s.afterglowPeriod + 10
```

Comment:
[nit] What happened to the spacing here? Seems like some auto-formatting might be playing some nasty tricks

Reply:
Looks like an interesting [quirk](https://github.com/golang/go/issues/1206) of gofmt. When I run `gofmt` directly over this file it doesn't report any difference, and it seems like it does remove spaces sometimes "to express binding strength".